merk/src/merk/restore.rs

```diff
@@ -286,8 +287,8 @@ impl<'db, S: StorageContext<'db>> Merk<S> {
     /// Creates a new `Restorer`, which can be used to verify chunk proofs to
     /// replicate an entire Merk tree. A new Merk instance will be initialized
     /// by creating a RocksDB at `path`.
-    pub fn restore(merk: Merk<S>, expected_root_key: Vec<u8>, expected_root_hash: CryptoHash) -> Restorer<S> {
-        Restorer::new(merk, expected_root_key, expected_root_hash)
```

Comment:
Why none?

Reply:
because chunking starts from the root tree and at that point, there is no combining value since it is not bounded by a tree element.